components/announcement_bar/freemium_trial_ended_announcement_bar/index.tsx

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import {FormattedMessage} from 'react-intl';
+import {useSelector, useDispatch} from 'react-redux';
+
+import {t} from 'utils/i18n';
+
+import AnnouncementBar from '../default_announcement_bar';
+import {AnnouncementBarTypes, Preferences, CloudBanners} from 'utils/constants';
+import useGetUsageDeltas from 'components/common/hooks/useGetUsageDeltas';
+import useGetLimits from 'components/common/hooks/useGetLimits';
+import {GlobalState} from 'types/store';
+import {isSystemAdmin} from 'mattermost-redux/utils/user_utils';
+import {makeGetCategory} from 'mattermost-redux/selectors/entities/preferences';
+import useGetSubscription from 'components/common/hooks/useGetSubscription';
+import {savePreferences} from 'mattermost-redux/actions/preferences';
+import {
+    getCurrentUser,
+} from 'mattermost-redux/selectors/entities/users';
+import {anyUsageDeltaValueIsNegative} from 'utils/utils';
```

Comment:
nit: can we group here by related paths (i.e. `mattermost-redux/`, `components/common` )?